ground/src/main/res/layout/map_type_dialog_item.xml

```diff
@@ -21,24 +21,24 @@
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="wrap_content"
   android:layout_height="wrap_content"
+  android:layout_marginLeft="4dp"
+  android:layout_marginRight="4dp"
   android:orientation="vertical">
 
-  <FrameLayout
-    android:id="@+id/container"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:padding="5dp">
-    <androidx.cardview.widget.CardView
-      android:layout_width="128dp"
-      android:layout_height="128dp"
-      app:cardCornerRadius="8dp">
-      <ImageView
-        android:id="@+id/imageview"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        tools:src="@drawable/ic_type_roadmap" />
-    </androidx.cardview.widget.CardView>
-  </FrameLayout>
+  <com.google.android.material.card.MaterialCardView
+    android:id="@+id/card"
+    android:layout_width="116dp"
+    android:layout_height="116dp"
```

Comment:
This looks amazing! The only visual glitch I see is it appears the right side of the satellite/hybrid thumbnail is clipped a bit. Is it possible to allow the cards to grown/shrink based on available container width, up to some max size?

Reply:
It's a scrollable list at the moment. I can try replacing it with a static view which shrinks/fills up the available container width with a max size of 116dp.

Reply:
I think that's preferable. The number of options will be fixed (3) for the foreseeable future, so no need for the user to scroll to catch the extra bit of thumbnail. Thanks!

Reply:
Hi Shobhit, I'm going to approve this PR in case you want to remove scrolling in a follow-up PR. HTH!

Reply:
I tried creating a static view but it requires a lot more changes than I anticipated (basically a rewrite).